BrowserKit/Sources/WebEngine/Utilities/URLFormatter.swift

```diff
@@ -16,10 +16,10 @@ public protocol URLFormatter {
     func getURL(entry: String) -> URL?
 }
 
-public class DefaultURLFormatter: URLFormatter {
```

Comment:
Please revert to make public again (the init as well). This is used under the `SampleBrowser.xcodeproj` app